bam_ampliconclip.c

```diff
@@ -111,21 +266,21 @@ int load_bed_file_pairs(char *infile, int get_strand, int sort_by_pos,
                 ret = 1;
                 goto error;
             }
-            if (*pairs->ref) {
-                if (strncmp(ref, pairs->ref, 256)) {
+            if (*entries->ref) {
+                if (strncmp(ref, entries->ref, 256)) {
                     fprintf(stderr, "[ampliconclip] error: "
```

Comment:
I'm unsure why this error message and check are still here.  It's meant to handle this case now afterall.

Is it a rebasing issue?  I think I added that check relatively recently, perhaps before you forked off your branch.

Reply:
For the record (see above) this is the old code needed to make ampliconstats build.  Once ampliconstats is converted we can remove this entire section of code.